README.md

```diff
@@ -4,32 +4,391 @@
 
 ## Description
 
-[Loki](https://grafana.com/oss/loki/) is an open-source fully-featured logging system. The Loki charmed operator deploys Loki in Kubernetes using [Juju](https://juju.is) and the [Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm).
+[Loki](https://grafana.com/oss/loki/) is an open-source fully-featured logging system. This Loki charmed operator handles installation, scaling, configuration, optimisation, networking, service mesh, observability, and Day 2 operations specific to [Loki](https://grafana.com/docs/loki/latest/) using [Juju](https://juju.is) and the [Charmed Operator Lifecycle Manager (OLM)](https://juju.is/docs/olm).
 
+On the principle that an operator should *"do one thing and do it well"*, this operator drives Loki application only. However, it can be composed with other operators to deliver a complex application or service. Because operators package expert knowledge in a reusable and shareable form, they hugely simplify software management and operations.
 
-## Quick start
 
-Create a Juju model named `observability`. This also sets the new model as the current one in the Juju command-line interface.
+
+## Getting started
+
+### Basic deployment
+
+Create a Juju model for your operator, say "observability"
 
 ```bash
 juju add-model observability
 ```
 
-Deploy the Loki Operator from Charmhub to the current model using the Juju command-line interface:
+The Loki Charmed Operator may be deployed using the Juju command line in a quite very way.
```

Comment:
Is there perhaps a typo in `in a quite very way`?

Reply:
Given the phrase `The way to relate Loki and Grafana is again very simple` lower in the document, perhaps the intention was to that Loki ... may be deployed ... in a simple way?